commentCounter.js

```diff
@@ -0,0 +1,3 @@
+export default function counterComment(item) {
+  return item.length;
+}
```

Comment:
- [ ] Still, functions in the comments JS logic is using the `function` keyword instead of the `arrow` function of the `es6` version of JS please update them.